falkordb_bulk_loader/relation_type.py

```
@@ -90,9 +90,7 @@ def process_entities(self):
                 try:
                     row_binary = struct.pack(fmt, src, dest) + self.pack_props(row)
                 except SchemaError as e:
-                    raise SchemaError(
-                        "%s:%d %s" % (self.infile.name, self.reader.line_num, str(e))
-                    )
+                    raise SchemaError("%s:%d %s" % (self.infile_name, idx, str(e)))
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)